### ISSUE TITLE
The 'version' parameter has a deprecated format 'canary'

### ISSUE THREAD

**Issue by alexisdechiara** [TryGhost/action-deploy-theme#79]
When i try to use action in my project i have the current issue : 

![image](https://user-images.githubusercontent.com/48689074/208914559-6b7337e4-ee14-4b58-a0e7-748bc8fa1520.png)

Here my main.yml file : 

```
name: Deploy Theme
on:
  push:
    branches:
      - master
      - main
jobs:
  deploy:
    runs-on: ubuntu-22.04
    steps:
      - uses: actions/checkout@v3.0.2
      - name: Deploy Ghost Theme
        uses: TryGhost/action-deploy-theme@v1.6.1
        with:
          api-url: ${{ secrets.GHOST_ADMIN_API_URL }}
          api-key: ${{ secrets.GHOST_ADMIN_API_KEY }}
```

**Comment by erinmikailstaples** [comment]
Getting the same error here on this which is completely causing the deployment to fail. 

Any thoughts on what might be causing this?

**Comment by mohab-sameh** [comment]
@alexisdechiara @erinmikailstaples Same issue here.
Seems to be an issue with the Ghost API. Did you find any workarounds?

**Comment by hcyildirim** [comment]
Same here, any updates?

**Comment by royalfig** [comment]
Fixed with #80!
